src/snippets/responsive-image.liquid

```diff
@@ -65,7 +65,7 @@
     {%- assign width_num = width | plus: 0 | round -%}
     {%- if image.width >= width_num -%}{{ width_num }},{%- endif -%}
   {%- endfor -%}
-  {{ image.width }}
+  {{- image.width -}}
```

Comment:
On our version of this file, I added in `at_most` to set an upper limit on the width.  Currently, we have it set to 4472.

`{{- image.width | at_most: 4472 -}}`

We have seen merchants upload giant images that are below the 20-megapixel limit but significantly wider than 4,472px.  For example even with the 4472 x 4472px limit someone can upload an image that's 8,944 x 2,236px.
https://help.shopify.com/en/manual/products/product-variant-images#image-sizes